### ISSUE TITLE
DGP E!=1?

### ISSUE THREAD

**Issue by karabelaselias** [joakimjohnander/DGPNet#5]
Hi there, I think your idea of using GP is very interesting, and I started to play around with your code together with a PhD student of mine. What was not clear for me from your arXiv article was the role of E. In your code it's fixed to one, however in your paper you write that you're not bound to a value of one.

My understanding of GPEs is, that for a single output feature your strategy works out fine, but if E!=1 then you would have a multioutput GP which could have correlations between the components of the output. Could you maybe explain the role of E in your paper and code?

**Comment by Parskatt** [comment]
Hi, I think we state in the paper that for multidimensional output we assume that the dimensions are uncorrelated. This means that the kernel becomes block diagonal, which simplifies computation. 
![image](https://github.com/joakimjohnander/DGPNet/assets/22053118/3a2ee852-544f-41ca-9ba2-afaa7fa15d31)
I think we had a bit more thorough explanation in a previous version of the paper.

**Comment by Parskatt** [comment]
Basically, we assume that the feature vector channels are uncorrelated over the image grid.

**Comment by Parskatt** [comment]
I don't think we explicitly investigated whether this holds true in practice in the paper, perhaps an interesting thing to look at :)

**Comment by karabelaselias** [comment]
Hi, thanks yes we thought you might have used those assumptions. Thanks for the fast reply.
